OData/src/System.Web.OData/OData/Formatter/Serialization/ODataCollectionSerializer.cs

```diff
@@ -52,13 +53,17 @@ public ODataCollectionSerializer(ODataSerializerProvider serializerProvider)
         public sealed override ODataValue CreateODataValue(object graph, IEdmTypeReference expectedType,
             ODataSerializerContext writeContext)
         {
-            if (graph == null)
+            var config = writeContext.Request.GetConfiguration();
```

Comment:
This only executes in the context of a request.  I could do so, but what would I throw - "this situation shouldn't be possible"? ;)